04_pig_latin/pig_latin.rb

```diff
@@ -0,0 +1,28 @@
+def foo(value, number)
```

Comment:
:)))) not a meaningful name. this is more like lorem ipsum. you've copy-pasted what i sent you lol :))